test/batch.coffee

```diff
@@ -0,0 +1,99 @@
+{ testBatch } = require './test'
```

Comment:
Can you add tests that do things like filters/expands/etc please

Reply:
I added a filter in one of the batch tests. I didn't add more as I wrote the tests in a way where it is easy to try different combinations of requests, but would result in a lot of duplication if we had to mirror the filter/expand/ecc tests. It is definitely good to have at least a couple, just to check if the compilation works properly in batch requests. I will try to refactor the tests a bit so that we have a couple of those as well.